src/test/java/org/opensearch/neuralsearch/processor/ScoreCombinationTechniqueTests.java

```diff
@@ -19,7 +21,7 @@ public class ScoreCombinationTechniqueTests extends OpenSearchTestCase {
 
     public void testEmptyResults_whenEmptyResultsAndDefaultMethod_thenNoProcessing() {
         ScoreCombiner scoreCombiner = new ScoreCombiner();
-        scoreCombiner.combineScores(List.of(), ScoreCombinationFactory.DEFAULT_METHOD);
+        scoreCombiner.combineScores(new CombineScoresDTO(List.of(), ScoreCombinationFactory.DEFAULT_METHOD, Collections.emptyList(), null));
```

Comment:
Please use build style, check my other comment in DTO class. With the init through constructor it's not better than having all arguments passed directly, we just slightly moved burden from one place to another